backend/src/test/java/com/woowacourse/levellog/team/domain/ParticipantsIngredientTest.java

```diff
@@ -0,0 +1,103 @@
+package com.woowacourse.levellog.team.domain;
```

Comment:
```suggestion
package com.woowacourse.levellog.domain;
```
domain 패키지 하위로 옮겨주세요 ^0^

Reply:
수정했습니다😄